Library/Formula/launchdns.rb

```diff
@@ -21,6 +21,7 @@ def install
   end
 
   test do
+    system "#{bin}/launchdns --version | grep -vq 'without socket activation'"
```

Comment:
Can’t we use something like `assert shell_output("#{bin}/launchdns --version") !~ /without socket activation/`?